src/include/stir/ProjDataInfo.h

```diff
@@ -348,9 +350,10 @@ class ProjDataInfo
   
 protected:
   virtual bool blindly_equals(const root_type * const) const = 0;
+//PW Shifted scanner_ptr from private to protected as it is being used in derived class for intrinsic tilt.
+  shared_ptr<Scanner> scanner_ptr;
```

Comment:
this isn't necessary. Let's use `get_scanner_ptr()` in the derived classes (like in most places already)